dev/reference/tools/tools-in-kubernetes.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Kubernetes 上的 TiDB 工具指南
+category: reference
+---
+
+# Kubernetes 上的 TiDB 工具指南
+
+Kubernetes 上的 TiDB 运维管理需要使用一些开源工具，同时，在 Kubernetes 上使用 TiDB 生态工具时，也有特殊的操作要求。本文档详细描述 Kubernetes 上的 TiDB 相关的工具及其使用方法。
+
+## 在 Kubernetes 上使用 PD Control
+
+[PD Control](reference/tools/pd-control.md) 是 PD 的命令行工具，在使用 PD Control 操作 Kubernetes 上的 TiDB 集群时，需要先使用 `kubectl port-forward` 打开本地到 PD 服务的连接：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl port-forward -n <namespace> svc/<cluster-name>-pd 2379:2379 &>/tmp/portforward-pd.log
+```
+
+执行上述命令后，就可以通过 `127.0.0.1:2379` 访问到 PD 服务，从而直接使用 `pd-ctl` 命令的默认参数执行操作，如：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+pd-ctl -d config show
+```
+
+假如你本地的 2379 被占据，则需要选择其它端口，此时需要为 `pd-ctl`  命令显式指定 PD 端口：
+
+```shell
+pd-ctl -u 127.0.0.1:<port> -d config show
+```
+
+## 在 Kubernetes 上使用 TiKV Control
+
+[TiKV Control](reference/tools/tikv-control.md) 是 TiKV 的命令行工具，在使用 TiKV Control 操作 Kubernetes 上的 TiDB 集群时，针对 TiKV Control 的不同操作形式，有不同的操作步骤。
+
+* **远程模式**：此模式下 `tikv-ctl` 命令需要通过网络访问 TiKV 服务或 PD 服务，因此需要先使用 `kubectl port-forward` 打开本地到 PD 服务以及目标 TiKV 节点的连接：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl port-forward -n <namespace> svc/<cluster-name>-pd 2379:2379 &>/tmp/portforward-pd.log
+    ```
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl port-forward -n <namespace> <tikv-pod-name> 20160:20160 &>/tmp/portforward-tikv.log
+    ```
+
+    打开连接后，即可通过本地的对应端口访问 PD 服务和 TiKV 节点：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    $ tikv-ctl --ca-path ca.pem --cert-path client.pem --key-path client-key.pem --host 127.0.0.1:20160 <subcommands>
```

Comment:
Currently, tidb-operator does not support TiKV data encryption and TiDB cluster inter-connection TLS. So the certificate is unnecessary.